core/shared/src/test/scala/cats/parse/ParserTest.scala

```diff
@@ -244,6 +244,25 @@ object ParserGen {
     val fn: A => F[B]
   }
 
+  def selected(ga: Gen[GenT[Parser]]): Gen[GenT[Parser]] =
+    Gen.zip(ga, pures).flatMap { case (parser, genRes) =>
+      val gpab = Gen.either(parser.fa, genRes.fa.map(Parser.pure)).map {
+        _ match {
+          case Left(pa) => pa.map(Left(_))
+          case Right(pb) => pb.map(Right(_))
+        }
+      }
```

Comment:
Feels like there is a $10 function I'm overlooking here.  I think Haskell calls it `uncozip`.

Reply:
Why the match? Why not just do the case at the top level?

Reply:
Because I've been locked down with these kids since March and am getting a little duller by the hour.